### DISCUSSION TITLE
Deploying With Vercel

### DISCUSSION THREAD

**Discussion by sfarrowGR** [keystonejs/keystone#7647] [Questions]
Has anyone been able to deploy to Vercel recently?  I have tried the boilerplate starters, and several on the example packages.  No matter which I try I can't seem to get a successful deploy.  I get the following error:

Error: No Output Directory named "public" found after the Build completed. You can configure the Output Directory in your Project Settings. Learn More: https://vercel.link/missing-public-directory

I have tried various output directory options such as .keystone .keystone/admin and ./. However, with these the app "builds" and runs but I just get page not found errors.

I feel like I must be missing something obvious.  Any help would be appreciated.  The same code deploys fine to Railway.

**Comment by diegovfeder** [comment]
I'm stuck in the same part, trying to deploy Keystone Admin UI to dev or prod.

**Reply by diegovfeder** [reply]
@sfarrowGR did you manage to make it work?

**Comment by abernh** [comment]
@borisno2 aka  Josh Calder states in [his video on deploying Keystone to Vercel](https://youtu.be/Wq6eQVyW8ug?t=247) to use 

```
.keystone/admin/.next
```
as the `output directory` value.

**Reply by abernh** [reply]
Then again there is a discussion on this topic in https://github.com/keystonejs/keystone/discussions/8254 that explains the intricacies of deploying to Vercel a little bit more and why it is unfortunately "not just as simple as using the correct value for the output directory.
